src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -597,7 +617,7 @@ private Result ConstructFormsWithoutAutocompletePreventionResult(ContrastLogRead
                     {
                         ArtifactLocation = new ArtifactLocation
                         {
-                            //UriBaseId = "RuntimeGenerated",
```

Comment:
>RuntimeGenerated [](start = 43, length = 16)

@mikefan@microsoft.com or @harleenkohli, whichever one of you wrote or commented this line -- _did_ you intend a different `uriBaseId` than `SITE_ROOT` here?

Reply:
No, I looked at the output from this rule. `SITE_ROOT` is correct.

---
In reply to: [289119180](https://github.com/microsoft/sarif-sdk/pull/1513#discussion_r289119180) [](ancestors = 289119180)